app/javascript/components/intro/components/list/component.js

```diff
@@ -1,47 +1,92 @@
-import React, { Fragment } from 'react';
+import React, { Fragment, useState } from 'react';
+import classnames from 'classnames';
 import Icon from 'components/icon';
+import videos from './constants';
 
 
 import './styles.scss';
 
-const List = () => (
-  <Fragment>
-    <div className="c-header-list">
-      <h1>Do you want to know more?</h1>
-      <ul>
-        <li className="list-item">
-          <Icon
-            name="play"
-            className="-small"
-          />
-          <a href="https://www.youtube.com/watch?v=BbNzIzmpRXg" target="_blank" rel="noopener noreferrer">
-            Financial service providers
-          </a>
-        </li>
-
-        <li className="list-item">
-          <Icon
-            name="play"
-            className="-small"
-          />
-          <a href="https://www.youtube.com/watch?v=pH9nwDL3Tfg&t=67s" target="_blank" rel="noopener noreferrer">
-            Regulators
-          </a>
-        </li>
-
-        <li className="list-item">
-          <Icon
-            name="play"
-            className="-small"
-          />
-          <a href="https://www.youtube.com/watch?v=K-0rCW0ijII&t=91s" target="_blank" rel="noopener noreferrer">
-            Insurance
-          </a>
-        </li>
-      </ul>
-    </div>
-  </Fragment>
-);
+const List = () => {
+  const [video, setState] = useState({
+    src: '',
+    title: '',
+    id: '',
+    isActive: true
+  });
 
+  const handleClick = (id) => {
+    const { src, title } = videos[id];
+
+    setState({
+      ...video,
```

Comment:
I don't think you need to duplicate the data in the state, I mean, if you have already a list of videos you should only save an "id" that says that this video is selected...